frontend/src/lib/components/DataDisplay/TableDisplay.svelte

```diff
@@ -65,12 +56,18 @@
 	hoverable={true}
 	striped={true}
 >
-	<TableHeader {caption} columns={Object.keys(data[0])} />
+	<TableHeader {caption} columns={Object.keys(data[0])} links={headerlinks} />
 	<TableBody tableBodyClass="divide-y">
-		{#each filteredItems as item}
+		{#each filteredItems as row, i}
 			<TableBodyRow>
-				{#each Object.entries(item) as pair}
-					<TableCell key={pair[0]} value={pair[1]} {statusIndicator} {statusColumns} />
+				{#each Object.entries(row) as pair, j}
+					<TableCell
+						key={pair[0]}
+						value={pair[1]}
+						{statusIndicator}
+						{statusColumns}
+						href={celllinks[i] ? celllinks[i][j] : ''}
```

Comment:
I think you can do `celllinks.?[i].?[j] : ''` here

Reply:
good point, used that.